react/components/ColorPicker/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ColorContent from './ColorContent'
+
+import './color-picker.global.css'
+
+/**
+ * ColorPicker App
+ */
+export default class ColorPicker extends React.Component {
+  /** Render ColorPicker App */
+  render() {
+    return <ColorContent {...this.props} />
```

Comment:
Tirar indereção trocando a implementação desse componente pra ser o proprio `ColorContent`

Se isso for feito a ferramenta de documentação consegue fazer analise estatica das propTypes que foram definidas e colocar na documentacao do componente.